docs/reference/koswat_cost_report.md

```diff
@@ -41,6 +41,7 @@ acceptance
 |   |   |   |   summary_costs.csv
 |   |   |   |   summary_locations.csv
 |   |   |   |   Stabiliteitswand.png
+|   |   |   |   Verticale piping oplossing.png
```

Comment:
don't u se sapces for images, underscores instead. Also, where's the image?

Reply:
I adjusted the `__str__` method to replace spaces by underscores and moved it to class `ReinforcementProfile`, so it is applied for all reinforcements, including `Grondmaatregel profiel`.

Reply:
I still want to keep names of files with underscores rather than spaces please change it accordingly.

Reply:
That's, in fact, what I've changed here. Please let me know if I overlooked something.

Reply:
Discussed: keep __str__ without replacing " " by "_". That is the responsibility of the exporter.